specification/window.browser.bs

```diff
@@ -52,6 +52,39 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     it MUST be used only for WebExtension related functionality.
   </section>
 
+  <section>
+    For backwards compatibility with older versions of web extensions, a UA MAY
+    alias individual properties of {{browser}} to <code>chrome</code>. {{browser}}
+    and <code>chrome</code> SHOULD NOT be direct aliases. Instead, individual
+    subproperties SHOULD be what is alaised. Any modification of a property of
+    {{browser}} SHOULD be reflected on <code>chrome</code>, and vice versa.
```

Comment:
Should we remove this last line? I read it as "if I modify a property on browser, it should be reflected on chrome", i.e exactly what we don't want to support. Or we could try to clarify it, but given the detailed example below, I'm not sure we need to.